plugins/CoreHome/vue/src/SiteSelector/SitesStore.ts

```diff
@@ -50,19 +56,39 @@ class SitesStore {
     onlySitesWithAtLeastWriteAccess = false,
     siteTypesToExclude: string[] = [],
   ): Promise<DeepReadonly<Site[]>|null> {
-    if (this.state.isInitialized && sitesToExclude.length === 0) {
+    if (
```

Comment:
I think we need at least some more changes to get this working without moving around issues.

The copy dialog site selector uses the same store as the global dashboard site selector. If we apply this change the global store will change once the copy dialog was opened but won't reset after it was closed. Thus, limiting the sites available in the global selector afterwards.

Maybe we can get around this by converting the site selector store from a singleton instance to an "instance per selector". That is, if `QuickAccess` and `TagManager.CopyDialog` play nice with that 🤔

Reply:
@mneudert Did you see an issue while testing these changes? When I test, the global site selector continues to work as expected since its filters don't match those of the copy modal. The whole point of my changes is so that the singleton uses the cached list when filters are constant and searches when the filters change. From my testing, the only filter which was being applied to the default list was the `sitesToExclude`, so this also fixes a previous bug where the `onlySitesWithAdminAccess` filter was only being honoured during text searches.

Reply:
Thank you @mneudert . The videos and your explanation were very helpful. I'll look into a fix for `hasMultipleSites()`.

Reply:
@mneudert I believe that was able to fix the issue by adding a little more to the store and then slightly updating the selector. When I test now, the `hasMultipleSites()` check works as expected in the global selector.